composeApp/src/androidMain/kotlin/com/swmansion/routinetracker/RoutineTrackerApplication.kt

```diff
@@ -1,8 +1,8 @@
 package com.swmansion.routinetracker
 
 import android.app.Application
-import com.swmansion.routinetracker.di.AppContainer
+import com.swmansion.routinetracker.di.DefaultAppContainer
 
 class RoutineTrackerApplication : Application() {
-    val appContainer: AppContainer by lazy { AppContainer(this) }
+    val defaultAppContainer: DefaultAppContainer by lazy { DefaultAppContainer(this) }
```

Comment:
Let’s keep the property name and type unchanged; no need to expose implementation specifics.